src/Components/Characterdetails.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable */
+import { useSelector } from 'react-redux';
+import { BsBackspace } from 'react-icons/bs';
+import { Link } from 'react-router-dom';
+import Characters from './Characters';
+
+function Characterdetails() {
+  const pageURL = window.location.href;
+  const urlArray = pageURL.split('/');
+  const pageID = parseInt(urlArray[urlArray.length - 1], 10);
+  const characters = useSelector((state) => (state.charactersReducer));
+  const trueCharacter = (characters.filter((character) => character.id === pageID))[0];
+  return (
+    <div className="char-details-bg">
+      <div className="char-details">
+        <div className="detail-image">
+          <div>
+            <img src={trueCharacter.image} alt={trueCharacter.image} />
+          </div>
+          <div className="go-back">
+            <Link to="/characters">
+              <BsBackspace>
+                <Characters />
+              </BsBackspace>
+            </Link>
+          </div>
+        </div>
+        <div className="detail-desc">
+          <h1>{trueCharacter.name}</h1>
+          <ul className="detail-ul">
+            <li>
+              Status:
+              {'  '}
+              {trueCharacter.status === 'Alive'
+                ? (
+                  <span>
+                    {' '}
+                    {trueCharacter.status}
```

Comment:
[OPTIONAL] Here it's another good example where you could apply restructuring and avoid a lot of coding repetition. Read more about the [best practices of JavaScript here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)